tools/build/workspace.bzl

```diff
@@ -278,6 +278,12 @@ def gapid_dependencies(android = True, mingw = True, locals = {}):
             locals = locals,
         )
 
+        maybe_repository(
+            android_native_vulkan_validation_layers,
```

Comment:
I would just call the rule and the repository `ndk_vk_validation_layers`

Reply:
done